core/src/test/java/io/grpc/internal/ManagedChannelImplTest2.java

```diff
@@ -606,7 +606,7 @@ public void callsAndShutdownAndShutdownNow() {
   }
 
   private void subtestCallsAndShutdown(boolean shutdownNow, boolean shutdownNowAfterShutdown) {
-    FakeNameResolverFactory nameResolverFactory =
+    ManagedChannelImplTest2.FakeNameResolverFactory nameResolverFactory =
```

Comment:
Instead of changing all of these names, could we instead just change the one _other_ name. It looks like duplicate name is because of the `class FakeNameResolverFactory` on line 3743. Let's just change that one class's name. Worst-case, name it `FakeNameResolverFactory2`.

Ditto on line 3452 in ManagedChannelImplTest.java

Reply:
That's a nice idea. Updated.